tools/clang/test/HLSLFileCheck/d3dreflect/rdat_mintarget/sm69_barriers.hlsl

```diff
@@ -0,0 +1,156 @@
+// RUN: %dxilver 1.9 | %dxc -T lib_6_9 %s | %D3DReflect %s | %FileCheck %s -check-prefixes=RDAT
+
+// Check that stage flags are set correctly still for different barrier modes in SM 6.9.
+
+// RDAT: FunctionTable[{{.*}}] = {
+
+RWByteAddressBuffer BAB : register(u1, space0);
+
+// RDAT-LABEL: UnmangledName: "node_barrier"
+// RDAT: FeatureInfo1: 0
+// RDAT: FeatureInfo2: (Opt_RequiresGroup)
+// RDAT: ShaderStageFlag: (Node)
+// RDAT: MinShaderTarget: 0xf0068
+
+[shader("node")]
+[NodeLaunch("broadcasting")]
+[NodeDispatchGrid(1, 1, 1)]
+[NumThreads(1,1,1)]
+void node_barrier() {
+  GroupMemoryBarrierWithGroupSync();
```

Comment:
I don't think we need to keep all the ones copied from, and already tested in, the other file for SM 6.8.

Reply:
Removed redundant tests